packages/fleather/lib/src/widgets/text_line.dart

```diff
@@ -136,6 +137,7 @@ class _TextLineState extends State<TextLine> {
         text: text,
         textAlign: textAlign,
         strutStyle: strutStyle,
+        textWidthBasis: editorState.widget.textWidthBasis,
```

Comment:
I think it would be better to provide the `textWidthBasis` as a constructor parameter, rather than the editor state.